### ISSUE TITLE
Request: When uploading/downloading files, show how many files (out of how many files) have already been processed and currently processed file

### ISSUE THREAD

**Issue by KUNGERMOoN** [radimitrov/CSharpShellApp#322]
Basically title
It would be useful for anyone who tries to download/upload big folders - current implementation doesn't show any progress, and thus doesn't let the users to estimate how long the download might take, or wether downloading got stuck on some file, and if yes, which file is it.

**Comment by radimitrov** [comment]
Yes, that is good idea. Though it will likely need to be a file count based progress so time estimation won't be the best.
